spec/factories/green_flag/feature_event.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :green_flag_feature_event, class: GreenFlag::FeatureEvent do
+    event_type_code 'test_event'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.